draft-ietf-mls-architecture.md

```diff
@@ -1564,36 +1564,37 @@ emitted credentials might be compromised.
 #### Authentication compromise: Ghost users and impersonations
 
 One important feature of MLS is that all
-clients know who is in the group at all times. This means that - if all Members
-of the group and the Authentication Service are honest - no other parties than
+Members know which other clients are in the group at all times. If all Members
+of the group and the Authentication Service are honest, no parties other than
 the members of the current group can read and write messages protected by the
-protocol for that Group. Note that when a `basic` Credential is used, the
-Authentication Service also needs an out-of-band mechanism to verify the
-identity asserted in the Credential.
-
-Beware though, the link between the cryptographic identity of the Client and the
-real identity of the User is important. With some Authentication Service
-designs, a private or centralized authority can be trusted to generate or
-validate signature keypairs used in the MLS protocol. This is typically the
-case in some of the biggest messaging infrastructures.
+protocol for that Group.
 
+Beware though, the link between the cryptographic
+identity of the client and the real identity of the End User is important.
```

Comment:
I would actually just remove this sentence, which doesn't seem helpful.

Reply:
I agree that it's not clear what the reader should do with this information. At the same time this is at the heart of proper e2e authentication in person-to-person messaging. Maybe something that would better fit in the authentication service section?

Reply:
This is already in the AS section so I'll remove the sentence.